src/azure-cli/azure/cli/command_modules/cosmosdb/tests/latest/test_cosmosdb_commands.py

```diff
@@ -530,32 +531,6 @@ def test_cosmosdb_sql_container(self, resource_group):
         container_list = self.cmd('az cosmosdb sql container list -g {rg} -a {acc} -d {db_name}').get_output_in_json()
         assert len(container_list) == 0
 
-    @ResourceGroupPreparer(name_prefix='cli_test_cosmosdb_sql_container')
-    def test_cosmosdb_sql_container_analytical_ttl(self, resource_group):
```

Comment:
why removing the test?

Reply:
This command is not included in the API version